src/pokedex_data_index.c

```diff
@@ -0,0 +1,38 @@
+#include "pokedex_data_index.h"
+
+#include <nitro.h>
+#include <string.h>
+
+#include "narc.h"
+
+static u32 pokedex_data_narc_index = NARC_INDEX_APPLICATION__ZUKANLIST__ZKN_DATA__ZUKAN_DATA;
+static u32 weight_message_bank_index = 707;
+static u32 height_message_bank_index = 709;
```

Comment:
**polish:** The rest of the repo typically:

1. Uses `camelCase`.
2. Prefixes static data symbols with `s`, e.g. `sWeightMessageBankIndex`.